docker/gcp_pudl_etl.sh

```diff
@@ -68,19 +64,25 @@ function copy_outputs_to_gcs() {
 }
 
 function copy_outputs_to_distribution_bucket() {
-    echo "Copying outputs to GCP distribution bucket"
-    gsutil -m -u $GCP_BILLING_PROJECT cp -r "$PUDL_OUTPUT/*" "gs://pudl.catalyst.coop/$GITHUB_REF"
-
-    echo "Copying outputs to AWS distribution bucket"
-    aws s3 cp "$PUDL_OUTPUT/" "s3://pudl.catalyst.coop/$GITHUB_REF" --recursive
+    # Only attempt to update outputs if we have a real value of GITHUB_REF
+    if [ -n "$GITHUB_REF" ]; then
```

Comment:
Only deploy or delete old files if we have a real git ref -- avoids accidentally blowing away the whole bucket if this variable ends up unset somehow.